packages/coinstac-api-server/src/data/resolvers.js

```diff
@@ -468,6 +492,19 @@ const resolvers = {
         (payload, variables) => (!variables.consortiumId || payload.consortiumId === variables.consortiumId)
       )
     },
+    /**
+     * Result subscription
+     * @param {object} payload
+     * @param {string} payload.resultId The consortium changed
+     * @param {object} variables
+     * @param {string} variables.resultId The consortium listened for
+     */
+    resultChanged: {
```

Comment:
Not sure if we're going to ultimately need this as I'll already have a watch on the run in order to kick of client pipelines. Still, we'll see how that goes.